website/static/css/main.css

```diff
@@ -1,30 +1,68 @@
 .fa{
 	color: #555;
```

Comment:
Replace tab with 4 spaces. Also, look for indentation with 4 spaces in this entire file. I can see several places that has to be fixed.